docs/zh-cn/1.3.5/user_doc/skywalking-agent-deployment.md

```diff
@@ -0,0 +1,59 @@
+
+dolphinscheduler-skywalking
+=============================
+
+dolphinscheduler-skywalking 模块为 Dolphinscheduler 项目提供了 [Skywalking](https://skywalking.apache.org/) 监控代理。
+
+本文档介绍了如何通过此模块接入 Skywalking 8.4+。
+
+# 安装
+
+以下配置用于启用 Skywalking agent。
+
+### 通过配置环境变量 (使用 Docker 部署时)
+
+```shell
+
+docker run -d --name dolphinscheduler \
+    -e SKYWALKING_ENABLE=true \
+    -e SW_AGENT_COLLECTOR_BACKEND_SERVICES="your.skywalking-oap-server.com:11800" \
+    -e SW_GRPC_LOG_SERVER_HOST="your.skywalking-log-reporter.com" \
+    -e SW_GRPC_LOG_SERVER_PORT="11800"
+    ...
+    apache/dolphinscheduler:latest all
+
+```
+
+### 通过配置 install_config.conf (使用 Dolphinscheduler install.sh 部署时)
+
+添加以下配置到 `${workDir}/conf/config/install_config.conf`.
+
+```properties
+
+# skywalking config
+# note: enable skywalking tracking plugin
+enableSkywalking="true"
+# note: configure skywalking backend service address
+skywalkingServers="your.skywalking-oap-server.com:11800"
```

Comment:
Good job!Hi,what is different form skywalkingServers and skywalkingLogReporterHost?If they are the same host ,is it ok?

Reply:
```text
skywalkingServers  ==>  collector.backend_service
skywalkingLogReporterHost  ==>  plugin.toolkit.log.grpc.reporter.server_host
skywalkingLogReporterPort  ==>  plugin.toolkit.log.grpc.reporter.server_port
```

The configuration items are from [skywalking docs](https://github.com/apache/skywalking/tree/master/docs/en/setup/service-agent/java-agent).